includes/functions/functions_prices.php

```diff
@@ -432,7 +432,7 @@ function zen_get_products_base_price($product_id)
             WHERE products_id = " . (int)$product_id . "
             AND attributes_display_only != 1
             AND attributes_price_base_included=1
-            ORDER BY options_id, value";
+            ORDER BY options_id, CAST(value AS SIGNED)";
```

Comment:
This truncates decimals such that numbers like 0.1, 0.2, and/or 0.3 would all look the same: 0